### ISSUE TITLE
Ideas on Mosquito-Taxon-Classifier

### ISSUE THREAD

**Issue by PrakharRaghuwanshi** [malariagen/gsoc-mosquito-taxon-classifier#6]
Hello team, 
I have gone through the project details and I have tried to identify a few ways in which the project can be approached for optimizing efficiency and robustness:

1. Genomic-Based enhancements can be made via identifying key species markers if possible and prioritize morphologically important part of the sequence for better feature selection.
2. Computational optimizations such as dimensionality reduction can be used where it is applicable to improve efficiency, while also preserving classification power.
3. Further the Random forest model can be optimized using hyperparameter tuning and then models like XGBoost/LightGBM can be explored for better accuracy if feasible.
4. Also, the imbalanced data can be handled by applying oversampling techniques or class weighting to improve precision.

Please let me know if this is approach aligns with what you expect from the contributors and the overall deliverables of the project, or if it needs adjustments. 

Looking forward to hear from you!
Prakhar Raghuwanshi

**Comment by jonbrenas** [comment]
Thank you @PrakharRaghuwanshi. I think your ideas are perfectly in line with this project's expectations.

**Comment by PrakharRaghuwanshi** [comment]
Thanks @jonbrenas !! This will form the basis for my proposal.

**Comment by PrakharRaghuwanshi** [comment]
Hi @jonbrenas , @ahernank and team
I have mailed the first draft of my proposal based on my initial ideas and further research, I request you to please go through the doc and give your remarks on any underlying shortcomings, potential issues and scope of improvements in the proposal. I have also referred to the comments and suggestions by different contributors on the repository and the remarks of Mentors on those comments for drafting my proposal.

Hope to hear from you !!
